src/main/java/com/epam/reportportal/junit/ReportPortalListener.java

```diff
@@ -582,6 +593,12 @@ protected void stopTestMethod(@Nonnull final Object runner, @Nonnull final Frame
 	protected void stopTestMethod(@Nonnull final Object runner, @Nonnull final FrameworkMethod method,
 			@Nonnull final ReflectiveCallable callable, @Nonnull final ItemStatus status, @Nullable final Throwable throwable) {
 		FinishTestItemRQ rq = buildFinishStepRq(runner, method, callable, status);
+		if (status != ItemStatus.PASSED && throwable != null) {
+			rq.setDescription(String.format(DESCRIPTION_TEST_ERROR_FORMAT,
```

Comment:
Please don't set anything into requests outside "build" methods.

Reply:
Moved it to the build method.